### PR TITLE
add zap to mechvibes

### PR DESCRIPTION
**Pull request by spinofdoom**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/mechvibes.rb [hunk 1]
```diff
@@ -14,4 +14,11 @@
   end
 
   app "Mechvibes.app"
+
+  zap trash: [
+    "~/Library/Application Support/Mechvibes",
+    "~/Library/Preferences/com.electron.mechvibes.plist",
+    "~/Library/Saved Application State/com.electron.mechvibes.savedState",
+    "~/mechvibes_custom",
```
Comment (bevanjkay) [inline Casks/mechvibes.rb:22]: @spinofdoom Are you able to verify what files are stored in the `mechvibes_custom` folder?

I would like to confirm that the following won't be deleted;
-  `Files created by the user directly.`
See https://docs.brew.sh/Cask-Cookbook#stanza-zap

Comment (spinofdoom) [reply Casks/mechvibes.rb:22]: It's an empty directory: 
`$ ls -A ~/mechvibes_custom`
`$`

Comment (neutric) [reply Casks/mechvibes.rb:22]: Empty here means "to be filled by user". This directory holds [custom sound packs added by the user](https://mechvibes.com/index.php/soundpacks/), so shouldn't be zapped.

**Review by miccal — CHANGES_REQUESTED** [review]

**Comment by miccal** [comment]
Thank you @spinofdoom.

**Review by miccal — APPROVED** [review]

**Pull request merged**